src/Component/routes.php

```diff
@@ -31,7 +33,15 @@ public static function loadComs()
         $path = '../App/Components/';
         $disk_scan = array_diff(scandir($path), ['..', '.']);
         foreach ($disk_scan as $scans) {
-            require_once '../App/Components/'.$scans.'/routes.php';
+            $configFile = route()->com.$scans.'/component.json';
```

Comment:
The `$disk_scane` and `$configFile` are the different variables declaration.

And it should be concerned.

Reply:
yes, i think the second approach is best 
like ```configFile``` is that right?

Reply:
Yes. The second approach is better than first.

And other codes have same issue.

Reply:
yes thanks

no issue i dont know why this website show the issues
https://app.codacy.com/app/Lablnet/Zest_Framework/pullRequest?prid=3213058